### ISSUE TITLE
POROUS OBST

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#488]
```
Please complete the following lines...

Application Version: 5.1.1
SVN Revision Number:
Compile Date:
Operating System:windows

Describe details of the issue below
HI 
FOR SOME REASON SMOKE DOES NOT COME OUT FROM THE ROOM THROUGH THE 
POROUS OBST , THE FILE IS RUNNING in fds 5.1.1 !!
I WILL APPRECIATE IF YOU COULD ASSIST IN FINDING MY MISTAKE 

ALSO, WHEN THERE'S A HOLE INSTEAD OF POROUS OBST, SMOKE DOES NOT GET OUT 
OF THE DOMAIN, EVEN THOUGH ALL BOUNDARY VENTS ARE OPEN
DO YOU HAVE AN IDEA CONCERNING THIS ?

THANKS 
J 


THANKS
```

Original issue reported on code.google.com by `putoutthefirenow` on 2008-10-28 19:48:41

<hr>
- _Attachment: [KUB.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-509/comment-0/KUB.fds)_

**Comment by gforney** [comment]
```
We'll look at the case. In the meanwhile, try running with 5.2.3, the latest version

of FDS.
```

Original issue reported on code.google.com by `mcgratta` on 2008-10-28 19:59:37
- **Labels added**: OpSys-All, Performance

**Comment by gforney** [comment]
```
This bug was already fixed by drjfloyd. Wait for FDS 5.2.4, which should come out 
later this week.
```

Original issue reported on code.google.com by `mcgratta` on 2008-10-28 20:09:16

**Comment by gforney** [comment]
```
Couldn't repeat the second portion of your issue. If I delete the POROUS line and 
just leave the HOLE, smoke leaves the hole and exits the domain.  This is hard to 
see with the smoke3d display since the smoke is diffuse, but adding a SLCF of 
VISIBILTIY makes it clear.
```

Original issue reported on code.google.com by `drjfloyd` on 2008-10-28 20:43:39
- **Labels added**: Application-FDS

**Comment by gforney** [comment]
```
Hello
Do you expect FDS version 5.2.4 to be ready this week ?

thank you
J
```

Original issue reported on code.google.com by `putoutthefirenow` on 2008-11-02 12:42:57

**Comment by gforney** [comment]
```
Yes, I will post a new version Monday or Tuesday, Nov 3 or 4.
```

Original issue reported on code.google.com by `mcgratta` on 2008-11-02 16:33:26

**Comment by gforney** [comment]
```
We have run into a few additional problems, and we will not have a release until 
next week.
```

Original issue reported on code.google.com by `mcgratta` on 2008-11-07 19:02:48

**Comment by gforney** [comment]
```
I appreciate your comment since I have been following the new releases.
```

Original issue reported on code.google.com by `putoutthefirenow` on 2008-11-10 07:52:34

**Comment by gforney** [comment]
```
Have you tested the fix since the release of 5.2.4?  If so please report back if the
fix has addressed your issue.
```

Original issue reported on code.google.com by `drjfloyd` on 2008-12-03 14:10:13

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `drjfloyd` on 2009-03-13 15:29:32
